rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -5446,6 +5446,79 @@ public void getFastBridgeWallet_OK() {
         assertEquals(obtainedWallet.getBalance(), Coin.COIN);
     }
 
+    @Test
+    public void createFastBridgeFederationData_Ok() {
+        BridgeSupport bridgeSupport = spy(getBridgeSupport(bridgeConstants, mock(BridgeStorageProvider.class)));
+
+        Sha256Hash derivationArgumentsHash = Sha256Hash.wrap(HashUtil.randomHash());
+        Address userRefundAddress = mock(Address.class);
+        Address lpBtcAddress = mock(Address.class);
+        RskAddress lbcAddress = mock(RskAddress.class);
+
+        Sha256Hash resultDerivationHash = Sha256Hash.wrap(HashUtil.randomHash());
+        doReturn(resultDerivationHash).when(bridgeSupport).getFastBridgeDerivationHash(
+                derivationArgumentsHash,
+                userRefundAddress,
+                lpBtcAddress,
+                lbcAddress
+        );
+
+        FastBridgeFederationData fastBridgeFederationData = bridgeSupport.createFastBridgeFederationData(
+                derivationArgumentsHash,
+                userRefundAddress,
+                lpBtcAddress,
+                lbcAddress
+        );
+        Script resultRedeemScript = RedeemScriptParser.createMultiSigFastBridgeRedeemScript(
+                bridgeSupport.getActiveFederation().getRedeemScript(),
+                resultDerivationHash
+        );
+
+        Assert.assertEquals(resultDerivationHash, fastBridgeFederationData.getDerivationArgumentsHash());
+        Assert.assertEquals(ScriptBuilder.createP2SHOutputScript(resultRedeemScript), fastBridgeFederationData.getFastBridgeScriptHash());
+    }
+
+    @Test(expected = Exception.class)
+    public void createFastBridgeFederationData_Error() {
+        BridgeSupport bridgeSupport = spy(getBridgeSupport(bridgeConstants, mock(BridgeStorageProvider.class)));
+
+        Sha256Hash derivationArgumentsHash = Sha256Hash.wrap(HashUtil.randomHash());
+        Address userRefundAddress = mock(Address.class);
+        Address lpBtcAddress = mock(Address.class);
+        RskAddress lbcAddress = mock(RskAddress.class);
+
+        Sha256Hash resultDerivationHash = Sha256Hash.wrap(HashUtil.randomHash());
+        doReturn(null).when(bridgeSupport).getFastBridgeDerivationHash(
+                derivationArgumentsHash,
+                userRefundAddress,
+                lpBtcAddress,
+                lbcAddress
+        );
+
+        Assert.fail();
```

Comment:
```suggestion
```
If the test doesn't triggers an exception it will fail, no need for the assert.fail